src/parquet/file/reader.h

```diff
@@ -34,6 +34,8 @@ struct RowGroupStatistics {
   int64_t num_values;
   int64_t null_count;
   int64_t distinct_count;
+  std::string min;
+  std::string max;
```

Comment:
I think your suggestion to make it `const std::string*` makes more sense. We can document that the `RowGroupStatistics` has to be deep copied if it is required past the lifetime of the `RowGroupReader`.